notifier.py

```diff
@@ -84,6 +84,13 @@ def list_diff(subscribers, owners):
     else:
       milestone_str = 'not yet assigned'
 
+	moz_steps = ''
```

Comment:
@ebidel This is the definition. Is there some other way I should have done this?

Reply:
I think you can just declare `moz_links  = ''` replace all uses of `moz_steps` with `moz_links`. IOW, you don't need two variables.

Reply:
I think indentation is off on these lines. The line of your first change should align with the `intro` var.

Reply:
Apologies. I didn't intend there to be two variables. I changed the name some point.